qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -104,7 +104,7 @@
             gagent_check_type = powerdown
             journal_file = /var/log/journal
             cmd_prepared_and_restart_journald = "mkdir -p ${journal_file} && chgrp systemd-journal ${journal_file} && chmod g+s ${journal_file} && systemctl restart systemd-journald"
-            cmd_query_log = "journalctl -e | grep -6 -i 'core-dump'"
+            cmd_query_log = "journalctl -e | grep -i 'core-dump' | grep -A 6 -i 'qemu-guest-agent.service'"
```

Comment:
Is this command supported in old kernel version?

Reply:
@fbq815  It is not related with the product version but some basic shell command,  should be ok with all.

Reply:
@yanan-fu thanks. after solving conflict. this patch gets ready to be merged. Could you please merge it or I have to ask others?